sdk/remoterendering/mixedreality-remoterendering/tests.yml

```diff
@@ -0,0 +1,10 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    PackageName: "@azure/mixedreality-remoterendering"
+    ServiceDirectory: remoterendering
+    Location: eastus2
+    MatrixFilters:
+      - DependencyVersion=^$
```

Comment:
Just to double check, was this line copy/pasted from another config or did you intend to add this filter (which will have the effect of excluding min/max version testing)?

Reply:
Yes. This was just copy and pasted without thought :) It's worth noting that the tests take a few minutes, so putting some constraints on the number of combinations test is a good idea. (Although I don't know whether this is the best way of doing so.)

Reply:
@benbp Can I confirm that the current situation (where testing is somewhat constrained) is acceptable, or whether you would recommend another approach? As I state above, I would not recommend that all combinations be tested.

Reply:
@MalcolmTyrrell sorry I missed this notification. I think this is fine, though if you want to further constrain it, there are other options like filtering on node version, operating system, etc.

All the tests combinations run in parallel. Is your concern about pipeline resource usage, or about PR validation time?

Reply:
Thanks. Resource usage is the concern here. For now, I'll go with the current state.